sysdig/internal/client/v2/onboarding.go

```diff
@@ -8,15 +8,17 @@ import (
 
 const (
 	onboardingTrustedIdentityPath         = "%s/api/secure/onboarding/v2/trustedIdentity?provider=%s"
+	onboardingTrustedAzureAppPath         = "%s/api/secure/onboarding/v2/trustedAzureApp?app=%s"
 	onboardingTenantExternaIDPath         = "%s/api/secure/onboarding/v2/externalID"
 	onboardingAgentlessScanningAssetsPath = "%s/api/secure/onboarding/v2/agentlessScanningAssets"
 )
 
 type OnboardingSecureInterface interface {
 	Base
 	GetTrustedCloudIdentitySecure(ctx context.Context, provider string) (string, error)
+	GetTrustedAzureAppSecure(ctx context.Context, app string) (map[string]string, error)
 	GetTenantExternalIDSecure(ctx context.Context) (string, error)
-	GetAgentlessScanningAssetsSecure(ctx context.Context) (map[string]interface{}, error)
+	GetAgentlessScanningAssetsSecure(ctx context.Context) (map[string]any, error)
```

Comment:
any reason for changing this return type?

Reply:
it's not really a change, it's the preferred nomenclature